### PR TITLE
CENM release 1.3.4

### PR DESCRIPTION
**Pull request by YashNabar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ Documentation on Corda Enterprise Network Manager (CENM) can be found at [CENM D
 | ------------- |---------------------|
 | 1.2           | git checkout v1.2   |
 | 1.3           | git checkout v1.3   |
```
Comment (szymonsztuka) [inline README.md:10]: I think going forward we are deleting old Docker imaged with log4j vulnerabilities, so it's ok to remove line 1.3.
Also as all code is in one branch, the best is to just add here changes for 1.3.4 (so skip 1.4.3 for now), merge, make tag and then follow up with https://github.com/corda/cenm-deployment/pull/15 to make 1.4.3 - merge make a tag, and then maybe merge latest 1.5.3 ( i know it's already released, but maybe the best is have this code as HEAD).

**Review by szymonsztuka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (YashNabar) [reply README.md:10]: Should we maintain separate branches for 1.3.x and 1.4.x instead, and have the latest (1.5.x) on master?

Comment (jmacmahonr3) [reply README.md:10]: I think so, and tag them appropriately but the the 1.5 release should always be on master

**Comment by YashNabar** [comment]
Replaced by #19

**Pull request closed without merging**